5-network/02-formdata/article.md

```diff
@@ -154,36 +154,36 @@ This example submits an image from `<canvas>`, along with some other fields, as
 </body>
 ```
 
-Please note how the image `Blob` is added:
+Note como la imagen `Blob` es agregada:
 
 ```js
 formData.append("image", imageBlob, "image.png");
 ```
 
-That's same as if there were `<input type="file" name="image">` in the form, and the visitor submitted a file named `"image.png"` (3rd argument) with the data `imageBlob` (2nd argument) from their filesystem.
+Se obtiene el mismo resultado tal como si se tuviese un campo `<input type="file" name="image">` en el formulario, y el usuario enviara un archivo con nombre `"image.png"` (3er argumento) con los datos `imageBlob` (2do argumento) desde su sistema de archivos.
 
-The server reads form data and the file, as if it were a regular form submission.
+El servidor lee el formulario `form-data` y el archivo tal como si de un formulario regular se tratara.
 
-## Summary
+## Resumen
 
-[FormData](https://xhr.spec.whatwg.org/#interface-formdata) objects are used to capture HTML form and submit it using `fetch` or another network method.
+[FormData](https://xhr.spec.whatwg.org/#interface-formdata) objetos utilizados para capturar un formulario HTML y enviarlo utilizando `fetch` u otro método de red.
 
-We can either create `new FormData(form)` from an HTML form, or create a object without a form at all, and then append fields with methods:
+Podemos de este modo utilizar `new FormData(form)` tal que from es un elemento formulario HTML, o crear un objeto sin un formulario en absoluto, y agregar los campos con los siguientes métodos:
 
-- `formData.append(name, value)`
-- `formData.append(name, blob, fileName)`
-- `formData.set(name, value)`
-- `formData.set(name, blob, fileName)`
+- `formData.append(nombre, valor)`
+- `formData.append(nombre, blob, nombreDeArchivo)`
+- `formData.set(nombre, valor)`
+- `formData.set(nombre, blob, nombreDeArchivo)`
 
-Let's note two peculiarities here:
+Nótese aquí dos particularidades:
 
-1. The `set` method removes fields with the same name, `append` doesn't. That's the only difference between them.
-2. To send a file, 3-argument syntax is needed, the last argument is a file name, that normally is taken from user filesystem for `<input type="file">`.
+1. El método `set` remueve campos con el mismo nombre, mientras que `append` no. Esta es la única diferencia entre estos dos metodos.
+2. Para enviar un archivo, se requiere de tres argumentos, el último argumento es el nombre del archivo, el cual normalmente es tomado desde el sistema de archivos del usuario por el `<input type="file">`.
 
-Other methods are:
+Otros métodos son:
 
-- `formData.delete(name)`
-- `formData.get(name)`
-- `formData.has(name)`
+- `formData.delete(nombre)`
+- `formData.get(nombre)`
+- `formData.has(nombre)`
 
-That's it!
+¡Esto es todo!
```

Comment:
that's all folks!
jeje
infancia de... mediados del siglo pasado

Reply:
Pa, que te dí trabajo che...

Reply:
@MaskeZen
Casi todas las revisiones son así, y ahora vino @maksumi a corregir al corrector jaja el impaciente.
Siempre pasa, nuestro cerebro es más inteligente y perezoso que nosotros y es dificil  autocorregirnos porque lee lo que cree que debe leer.